hokusai/commands/env.py

```diff
@@ -16,13 +16,10 @@ def delete_env(context, namespace=None):
 def get_env(context, environment, namespace=None):
   configmap = ConfigMap(context, namespace=namespace)
   configmap.load()
-  if environment:
-    for k, v in configmap.all():
-      if k in environment:
-        print_smart("%s=%s" % (k, v))
-  else:
-    for k, v in configmap.all():
-      print_smart("%s=%s" % (k, v))
+  all_configs = configmap.all()
+  env_vars = {k:v for k,v in all_configs if (environment and k in environment) or not environment }
```

Comment:
@izakp @saolsen is there a better way to do this? `all_configs` is a _generator_ so i couldn't call `sorted` on it, had to first iterate and create the variables and separately sort it.
This is not a big issue unless we have like a million environment variables, but if we have a million env variables we have other problems 😄

Reply:
Haha nice one - Overoptimiztion on my part?  We can easily change the `ConfigMap.all()` method to _not_ be a generator (and if we have a million env vars yes we have other problems) by refactoring its [current implementation](https://github.com/artsy/hokusai/blob/master/hokusai/services/configmap.py#L59) to 
```
return self.struct['data'].items()
```

So we just return a dict and not an iterator 😅 https://stackoverflow.com/questions/10458437/what-is-the-difference-between-dict-items-and-dict-iteritems

Reply:
Then you can easily do `sorted(configmap.all().keys())`

Reply:
good point! did it! 🙌